probes/hasDangerousWorkflowScriptInjection/impl.go

```diff
@@ -63,8 +63,8 @@ func Run(raw *checker.RawResults) ([]finding.Finding, string, error) {
 				LineStart: &e.File.Offset,
 				Snippet:   &e.File.Snippet,
 			})
-			patch := patch.GeneratePatch(e.File)
-			f.WithPatch(&patch)
+			finding_patch := patch.GeneratePatch(e.File)
+			f.WithPatch(&finding_patch)
```

Comment:
Is this relevant for this PR?

Reply:
Ah it was a linter fix. This was shadowing the package name. Not strictly related to PR, but I'd keep it here just to keep everything linted afterwareds

Reply:
Just a thing I've noticed now: I think the pattern here should be CamelCase instead of snaked case.

Reply:
Oh I think you're right! It's weird that the linter didn't notice that, though 😕